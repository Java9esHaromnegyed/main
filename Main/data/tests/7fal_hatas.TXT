loadArena testArena7.txt
initArena
tick 2