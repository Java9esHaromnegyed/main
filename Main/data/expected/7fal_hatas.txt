arenaLoaded src: testArena7.txt
robotAdded id:1 name: "player1" pos: [0; 0]
obstacleAdded type: WALL pos: [1; 0]
arenaInited
tick
robotStopped id: 1 speed: 0
robotMoved id: 1 pos: [0; 0]